### ISSUE TITLE
Spaces in file names (mission 11)

### ISSUE THREAD

**Issue by rlepigre** [phyver/GameShell#47]
I get the following on `master`:
```
~/Castle/Great_hall
[mission 11] $ ls
11357_tapestry_ 9    17411_tapestry_ 5	      3773_stag_head	standard_3
1260_suit_of_armour  26272_decorative_shield  6166_tapestry_ 2	standard_4
14289_tapestry_ 1    27267_tapestry_ 7	      6250_tapestry_ 8
14468_tapestry_ 6    28572_tapestry_10	      standard_1
15745_tapestry_ 3    30970_tapestry_ 4	      standard_2
```

**Comment by phyver** [comment]
oups...
see 80a52a4617098536351eae6c69f847abb0bd71cf

**Comment by rlepigre** [comment]
By the way, I don't know if you know, but if your commit message contains something like "Fixes #47" or similar (with any number of issue tags) then GitHub automatically closes the issue and references the commit. :)

**Comment by phyver** [comment]
Ah!
Thanks.
